### ISSUE TITLE
What does Int Pres in Cutter Position chart mean?

### ISSUE THREAD

**Issue by gojanpaolo** [gldd-se/PostDredge-Public#38]
![image](https://user-images.githubusercontent.com/16026608/72402446-f0613f00-3714-11ea-83b7-8b25eb7709c7.png)

Maybe Intermediate Pressure? But that sounds incomplete because Intermediate Pressure should be a pressure value. It is station/range points as what's plotted in the chart. What are we actually trying to represent with those points?

**Comment by rcriii42** [comment]
This is a true/false indicating whether the hofer valve was activated by a low pressure event.

**Comment by gojanpaolo** [comment]
Looking at the chart, there is another point series called "Hoffer Open" which I think is redundant to the "Int Pres" series.

The values for that series are from the Hoffer_Valve column in the MDB file. But the values are always 0 when I ran queries to all the MDB files to at least 2 projects for each of the 4 cutter dredges. So I think that column is obsolete.

We've added a similar series for "Int Pres" in PostDredge but we won't be adding the "Hoffer Open" for now.
